config/templates/tomcat/tomcat.json

```diff
@@ -4,7 +4,11 @@
     "!logs_path_localhost_access_log": "{{ localLogsPathPrefix | default('/apps_ux/logs') }}/{{ app | upper }}/{{ component | lower }}/localhost_access_log.*.log",
     "!logs_path_localhost": "{{ localLogsPathPrefix | default('/apps_ux/logs') }}/{{ app | upper }}/{{ component | lower }}/localhost.*.log",
     "!logs_path_catalina": "{{ localLogsPathPrefix | default('/apps_ux/logs') }}/{{ app | upper }}/{{ component | lower }}/catalina.*.log",
-    "!logs_path_app": "{{ localLogsPathPrefix | default('/apps_ux/logs') }}/{{ app | upper }}/{{ component | lower }}/{{ component | lower }}.log"
+    "!logs_path_app": "{{ localLogsPathPrefix | default('/apps_ux/logs') }}/{{ app | upper }}/{{ component | lower }}/{{ component | lower }}.log",
+    "!startTomcatParserRegex": "{{ startTomcatParserRegex | default('/^\\\\d{2}-\\\\w{3}-\\\\d{4}\\\\s\\\\d{2}:\\\\d{2}:\\\\d{2}.\\\\d{3}.*$/') }}",
```

Comment:
Can probably just use
`"startTomcatParserRegex": "/^\\d{2}-\\w{3}-\\d{4}\\s\\d{2}:\\d{2}:\\d{2}.\\d{3}.*$/"`

The `| default` usage above is too clever. It's inverted with a parameter with less precedence being used. Here, we just want the app definition to be able to overrule and it already will have more precedence.